src/app/routes/auth/containers/LockedScreen.js

```diff
@@ -1,48 +1,79 @@
+        }
+    }
+};
 
 export default class LockedScreen extends React.Component {
-  render() {
-    return (
-      <div id="extr-page">
-        <div id="main" role="main">
-          <form className="lockscreen animated flipInY" action="#/dashboard/analytics">
-            <div className="logo">
-              <h1 className="semi-bold"><img src="assets/img/logo-o.png" alt=""/> SmartAdmin</h1>
-            </div>
-            <div>
-              <img src="assets/img/avatars/sunny-big.png" alt="" width="120" height="120"/>
-              <div>
-                <h1><i className="fa fa-user fa-3x text-muted air air-top-right hidden-mobile"/>John Doe
-                  <small><i className="fa fa-lock text-muted"/> &nbsp;Locked</small>
-                </h1>
-                <p className="text-muted">
-                  <a href="mailto:simmons@smartadmin">john.doe@smartadmin.com</a>
-                </p>
 
-                <div className="input-group">
-                  <input className="form-control" type="password" placeholder="Password"/>
-                  <div className="input-group-btn">
-                    <button className="btn btn-primary" type="submit">
-                      <i className="fa fa-key"/>
-                    </button>
-                  </div>
-                </div>
-                <p className="no-margin margin-top-5">
-                  Logged as someone else? <a href="#/login"> Click here</a>
-                </p>
-              </div>
+    constructor() {
+        super();
+        // Initialize variables for submit password action method to keep form as a controlled component
+        this.formSubmit = this.submitPassword.bind(this);
+    }
 
-            </div>
-            <p className="font-xs margin-top-5">
-              Copyright SmartAdmin 2014-2020.
+    componentDidMount() {
+        // Initialize variables for access to form fields
+        this.password = ReactDOM.findDOMNode(this.refs.password);
+    }
+
+    submitPassword(event) {
+        event.preventDefault();
 
-            </p>
-          </form>
+        const formValid = this.password.value.length >= 8;
+        if (formValid) {
+            // Todo: Check user password here
+            // password value -> this.password.value
+        }
+    }
 
-        </div>
-      </div>
-    )
-  }
+    render() {
+        return (
+            <div id="extr-page">
+                <div id="main" role="main">
+                    <UiValidate options={validationOptions}>
+                        <form className="lockscreen animated flipInY" action="#/dashboard" onSubmit={this.formSubmit}>
+                            <div className="logo">
+                                <div><h1 className="logo-name text-center">TrackIt+</h1></div>
+                            </div>
+                            <div>
+                                <img src="assets/img/avatars/sunny-big.png" alt="" width="120" height="120"/>
```

Comment:
let's upload the generic avatars that we will use when a user doesn't have a profile picture. Do this for other views that use profile pictures.